src/genomic_benchmarks_qc/cli.py

```
@@ -0,0 +1,233 @@
+    """
+    try:
+        command(*args, **kwargs)
+    except typer.Exit:
+        raise
+    except Exception as exc:
+        # Failures inside the evaluation proper are logged twice at DEBUG, once
+        # here and once by `log_failures`. Worth it: the alternative is guessing
+        # whether the evaluation got far enough to have logged anything at all.
+        logging.debug("Command failed.", exc_info=True)
+        typer.echo(f"Error: {exc}", err=True)
+        raise typer.Exit(code=1)
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)